test_helpers/doer_mock.go

```diff
@@ -0,0 +1,35 @@
+package test_helpers
+
+import (
+	"bytes"
+
+	"github.com/tarantool/go-tarantool/v2"
+)
+
+type DoerResponse struct {
+	resp MockResponse
+	err  error
+}
+
+type MockDoer struct {
+	requests  []tarantool.Request
```

Comment:
The idea: to be able to compare incoming requests from a test. So it should be in the public API.
```suggestion
	Requests  []tarantool.Request
```